app/scripts/controllers/swaps.js

```diff
@@ -141,6 +165,9 @@ export default class SwapsController {
       this.setSwapsErrorKey('')
     }
 
+    // Ensure we have a quotes refresh rate
+    await this._setSwapsQuoteRefreshTime()
```

Comment:
Could this call be made in parallel with `_fetchTradesInfo`? Or is there an important reason to block on fetching the quote refresh time before fetching quotes?